site/includes/layouts/default.njk

```diff
@@ -16,6 +16,8 @@
 
     {# favicon #}
     <link rel="icon" type="image/png" href="{{ site.images.favicon }}">
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@tailwindcss/ui@latest/dist/tailwind-ui.min.css">
```

Comment:
I couldn't get the Tailwind UI plugin to play nice with our TailwindCSS setup. But once that's figured out, we can remove this.